plugins/lsp/language-servers/pylsp.nix

```diff
@@ -12,483 +12,490 @@ in
 {
   # All settings are documented here:
   # https://github.com/python-lsp/python-lsp-server/blob/develop/CONFIGURATION.md
```

Comment:
Can you move these comments down, to above the `settings = {` line?

Reply:
Done!